core/templates/components/forms/custom-forms-directives/thumbnail-uploader.component.ts

```diff
@@ -150,12 +150,15 @@ export class ThumbnailUploaderComponent implements OnInit, OnChanges {
     let entityType = this.contextService.getEntityType();
     let entityId = this.contextService.getEntityId();
     const result = this.assetsBackendApiService.postThumbnailFile(
-      resampledFile, this.tempImageName, entityType, entityId).toPromise();
+      resampledFile,
+      (this.tempImageName || ''),
+      (entityType || ''),
+      (entityId || '')).toPromise();
     result.then((data) => {
       this.editableThumbnailDataUrl = (
         this.imageUploadHelperService
           .getTrustedResourceUrlForThumbnailFilename(
-            data.filename, this.contextService.getEntityType(),
+            data.filename, (this.contextService.getEntityType() || ''),
```

Comment:
Why is this added?

Reply:
It is of the type ```string | undefined``` but here a string is passed. So, as a default value I added the empty string.

Reply:
It needs a string but ```getEntityType``` is of the type ```string | undefined```

Reply:
@vojtechjelinek I faced this issue too. Is there any better way?

Reply:
`''` is not a good replacement for `null`, you need to either remove the `null` altogether or allow it explicitly everywhere